2200008/5.py

```diff
@@ -0,0 +1,22 @@
+# 암호생성기
+from collections import deque
+
+while True:
+    try:
+        t = int(input())
+        code = deque(map(int, input().split()))
+        while True:
+            for i in range(1, 6):
+                num = code.popleft()-i
+                if num < 0:
+                    num = 0
+                code.append(num)
+                if code[7] == 0:
+                    break
+            if code[7] == 0:
+                    break
+
+        print(f'#{t}', end=' ')
+        print(*code)
+    except:
```

Comment:
덱 활용 아주 좋습니다! 💯
```suggestion
    except EOFError:
```
`except` 문은 발생하게 되는 에러를 적어주시는 것이 좋습니다! `EOFError`는 입력 받을 줄이 없는 데 `input` 함수를 호출한 경우 발생하는 에러입니다.